### ISSUE TITLE
More accurate aria-labels on steps

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#2377]
# Info

Reported by Roy Styan

Currently when using the Form Stepper when tabbing through the steps, steps are read as either "Checked" or "Not Checked". Statuses read are actually not always accurate (ie. it will read a checked status as Not Checked), in addition we have other statuses, Approved, Not touched, working on, etc... And to top it off, the actual label for each step is not read by the screen reader.

## Acceptance Criteria:

1. Statuses are actually read appropriately depending on the context
2. Labels for each step are actually read out by a screen reader

**Comment by ArakTaiRoth** [comment]
@twjeffery Add to Acceptance Criteria, what the steps should be read out as

**Comment by ArakTaiRoth** [comment]
The Form Stepper has been deprecated and is no longer receiving any updates
